tests/utils/test_atom_util.py

```diff
@@ -187,6 +187,10 @@ def test_update_members_from_pref():
     aux.update_members_from_preferences(pref)
     assert aux.atom is None
```

Comment:
Pref tagged members are updated if not None otherwise left alone. Otherwise we would need to assume how to create the instance.